x-pack/plugins/ml/public/application/util/dependency_cache.ts

```diff
@@ -22,6 +22,7 @@ import type {
 } from 'kibana/public';
 import type { DataPublicPluginStart } from 'src/plugins/data/public';
 import type { SharePluginStart } from 'src/plugins/share/public';
+import { FieldFormatsStart } from 'src/plugins/field_formats/public';
```

Comment:
Nit: Could be `import type { FieldFormatsStart } from 'src/plugins/field_formats/public';`.

Reply:
Changed on https://github.com/elastic/kibana/pull/121517/commits/5cd8b54d6123e5cbf91a85612ecdcf6165d0d460.